### ISSUE TITLE
Method arguments that are simple member accesses and multiple lines should be placed on single lines

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#834]
Methods arguments as in following example should be reported and proposed to place them on single lines as they represent a specific argument.

So, code like:
```c#
DoSomething(xyz
              .Abc);
```
should be fixed to:
```c#
DoSomething(xyz.Abc);
```
